### PR TITLE
CLI backup create

### PR DESCRIPTION
**Pull request by aswietlik**
#74 #88

### CODE REVIEW

File: cli/cmd/backup/create.go [hunk 1]
```diff
@@ -2,33 +2,76 @@ package backup
...
+be created according to cron-style time provided.
 msp backup create "my-backup" --cluster "my-cluster"`,
 	Run: func(cmd *cobra.Command, args []string) {
-		backup := "mysql-backup"
+		options := options.ExtractOptions(cmd)
+
+		backupName = clusterName
 		if len(args) >= 1 {
-			backup = args[0]
+			backupName = args[0]
```
Comment (earlgreyz) [inline cli/cmd/backup/create.go]: We should add a constraint that user can provide only one or zero arguments (`Args:` validator for backupCreateCmd)

File: cli/cmd/backup/create.go [hunk 2]
```diff
@@ -2,33 +2,76 @@ package backup
...
+		}
 	},
 }
 
 func init() {
 	Cmd.AddCommand(backupCreateCmd)
 
-	backupCreateCmd.Flags().StringVar(&clusterName, "cluster", "",
+	backupCreateCmd.Flags().StringVarP(&clusterName, "cluster", "c", "",
 		"name of cluster for which the backup is made")
```
Comment (earlgreyz) [inline cli/cmd/backup/create.go]: "name of the cluster to be backed up"

File: cli/cmd/backup/create.go [hunk 3]
```diff
@@ -2,33 +2,76 @@ package backup
...
+		backupName = clusterName
 		if len(args) >= 1 {
-			backup = args[0]
+			backupName = args[0]
 		}
 
-		//TODO: create backup logic
-		fmt.Println(backup)
-		fmt.Println("backup create--cluster called")
+		err := createMySQLBackup(options)
```
Comment (earlgreyz) [inline cli/cmd/backup/create.go]: Maybe add a flag for specifying a backup name if it's optional?

Comment (aswietlik) [reply cli/cmd/backup/create.go]: Done.

File: cli/cmd/backup/create.go (hunk 2 above)
Comment (aswietlik) [reply cli/cmd/backup/create.go]: Done.

File: cli/cmd/backup/create.go (hunk 1 above)
Comment (aswietlik) [reply cli/cmd/backup/create.go]: Done,

File: cli/cmd/backup/create.go [hunk 4]
```diff
@@ -2,33 +2,84 @@ package backup
...
 
-	backupCreateCmd.Flags().StringVar(&clusterName, "cluster", "",
-		"name of cluster for which the backup is made")
+	backupCreateCmd.Flags().StringVarP(&clusterName, "cluster", "c", "",
+		"name of the cluster to be backed up")
 	backupCreateCmd.MarkFlagRequired("cluster")
+	backupCreateCmd.Flags().StringVarP(&time, "time", "t", "", "cron-style time")
+	backupCreateCmd.MarkFlagRequired("time")
+	backupCreateCmd.Flags().StringVarP(&storage, "storage", "s", "1Gi", "storage value")
+	backupCreateCmd.Flags().StringVar(&storage, "name", "", "backup name")
```
Comment (pkelv) [inline cli/cmd/backup/create.go]: Two times storage (backupName missing)

File: cli/cmd/backup/create.go [hunk 5]
```diff
@@ -2,33 +2,84 @@ package backup
...
+)
 
 var backupCreateCmd = &cobra.Command{
 	Use:   "create [backup name]",
-	Short: "short description of backup create",
-	Long: `A longer description of backup create with examples:
+	Short: "Schedules a new MySQL backup",
+	Long: `Schedules a new MySQL backup for specified cluster. The backups will
+be created according to cron-style time provided.
 msp backup create "my-backup" --cluster "my-cluster"`,
```
Comment (pkelv) [inline cli/cmd/backup/create.go]: Change description if backup name is provided from flag

**Pull request merged**